MetagenomeUtils.spec

```diff
@@ -43,13 +48,22 @@ module MetagenomeUtils {
 
     /*
       input_ref: BinnedContig object reference
+
+      optional params:
+      not_save_to_shock: not saving result bin files to shock
     */
     typedef structure {
       string input_ref;
+      boolean not_save_to_shock;
```

Comment:
I think this might be a confusing parameter/parameter name.  Maybe we can just call it:
  save_to_shock, and have a default value set to 1.